### DISCUSSION TITLE
Info menu button doesn't work.

### DISCUSSION THREAD

**Discussion by simplect** [Jyers/Marlin#66] [Q&A] [answered]
Hi!

I just tried flashing the latest 1.1 default noprobe. I wanted to check the firmware version, but the info menu button doesn't seem to do anything. Is it just me? :)

**Comment by simplect** [comment]
Also keep up the good work :) And a Discord or some chat thingy might be nice for such small issues.

**Comment by Jyers** [comment]
Is it just the info menu on the main screen that is not working, or is the one in the control menu failing as well?

**Reply by gstevenson** [reply]
The Control menu one works fine for me

**Comment by gstevenson** [comment]
I've also got this - if I hit the info button on the home screen it just sits around for a while. First time I tried it I power cycled the printer to get it back to life, second time I noticed if I waited it did resume functionality

**Comment by Jyers** [comment]
Just flashed the latest default noprobe and was unable to recreate the issue. Are you still experiencing this?

**Reply by chevyman142000** [reply]
Same happens for me. Info in control menu works fine, but info button on home screen doesn't do anything.

**Comment by Jyers** [comment, marked as answer]
Is this still an issue with the latest build?

**Reply by gstevenson** [reply]
unfortunately I've got my bltouch installed now so the icon's no longer there for me to test... @simplect?

**Reply by simplect** [reply]
It works! :) Merci
